<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationCurveMain</class>
 <widget class="QWidget" name="CalibrationCurveMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1700</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1700</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>-1</y>
     <width>381</width>
     <height>881</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_up" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 255, 247);

background: #FFFFFF;
border-radius: 10px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">width: 48px;
height: 24px;
font-family: PingFang SC;
font-weight: 500;
font-size: 26px;
color: #666666;
line-height: 42px;</string>
         </property>
         <property name="text">
          <string>项目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>345</width>
           <height>62</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>345</width>
           <height>62</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 1px solid #DCDEE0;
    border-radius: 28px;
    font-family: &quot;PingFang SC Medium&quot;;
    font-size: 24px;
    padding-left: 10px;
    background-color: rgb(250, 251, 252);
	font-weight: 500;
    color: #000000;
}

QComboBox::drop-down {
    margin: 0px;
    padding-right: 20px;
    border: 0px;
}

QComboBox::down-arrow {
    image: url(:/static/PHOTOS/combo_up.png);
}

QComboBox::down-arrow:on {
    image: url(:/static/PHOTOS/combo_down.png);
}

QListView {
    margin: 0px;
    padding: 0px;
    border: 0px;
    border-radius: 10px;
    outline: 0px;
    font-size: 24px;
    color: #333333;
    background-color: rgb(250, 251, 252);
}

QListView::item {
    margin: 0px;
    padding: 10px;
    border: 0px;
    border-radius: 0px;
    height: 60px;
}

QListView::item:hover {
    color: #333333;
}

QListView::item:selected {
    color: rgb(31, 112, 227);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">width: 102px;
height: 25px;
font-family: PingFang SC;
font-weight: 500;
font-size: 26px;
color: #666666;
line-height: 42px;</string>
         </property>
         <property name="text">
          <string>试剂批号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_batch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>345</width>
           <height>62</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>345</width>
           <height>62</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 1px solid #DCDEE0;
    border-radius: 28px;
    font-family: &quot;PingFang SC Medium&quot;;
    font-size: 24px;
    padding-left: 10px;
    background-color: rgb(250, 251, 252);
	font-weight: 500;
    color: #000000;
}

QComboBox::drop-down {
    margin: 0px;
    padding-right: 20px;
    border: 0px;
}

QComboBox::down-arrow {
    image: url(:/static/PHOTOS/combo_up.png);
}

QComboBox::down-arrow:on {
    image: url(:/static/PHOTOS/combo_down.png);
}

QListView {
    margin: 0px;
    padding: 0px;
    border: 0px;
    border-radius: 10px;
    outline: 0px;
    font-size: 24px;
    color: #333333;
    background-color: rgb(250, 251, 252);
}

QListView::item {
    margin: 0px;
    padding: 10px;
    border: 0px;
    border-radius: 0px;
    height: 60px;
}

QListView::item:hover {
    color: #333333;
}

QListView::item:selected {
    color: rgb(31, 112, 227);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_down" native="true">
      <property name="styleSheet">
       <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">background: #FFFFFF;
border-radius: 10px;

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>1271</width>
     <height>651</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1271</width>
     <height>651</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1271</width>
     <height>651</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>660</y>
     <width>1271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_2{
		background: #FFFFFF;
		border-radius: 10px;
}
</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
    </property>
    <property name="text">
     <string>主曲线:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
    </property>
    <property name="text">
     <string>校准曲线:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
    </property>
    <property name="text">
     <string>校准点:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_m_cur_show">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>44</y>
      <width>101</width>
      <height>6</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_m_cur_show{
	border-radius: 3px;
	background-color: rgb(0, 135, 209);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_m_cur_show_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>45</y>
      <width>101</width>
      <height>6</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_m_cur_show_2{
	border-radius: 3px;
	background-color: rgb(255, 204, 64);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_m_cur_show_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>38</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_m_cur_show_3{
	
	border-radius: 10px;
	background-color: rgb(0, 0, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
